### PR TITLE
Add isBrowserRenderingPaused

### LINKED ISSUE #1996: Add isBrowserRenderingPaused
**Is your feature request related to a problem? Please describe.**
There's no counterpart for [setBrowserRenderingPaused](https://wiki.multitheftauto.com/wiki/SetBrowserRenderingPaused)
so you can't check if the browser rendering is paused or not

**Describe the solution you'd like**
Add a new function called `isBrowserRenderingPaused` to check if the browser rendering is paused or not

**Describe alternatives you've considered**
None

### PR DESCRIPTION
**Pull request by fresholia**
Resolves #1996

```lua
isBrowserRenderingPaused(browser)
browser:isRenderingPaused

browser.renderingPaused
```

### CODE REVIEW

**Comment by Unde-R** [comment]
try to use the new parser

**Comment by Lpsd** [comment]
I don't think you should add this to `m_Properties`. Store it under `m_bIsRenderingPaused` instead.

**Comment by fresholia** [comment]
> I don't think you should add this to `m_Properties`. Store it under `m_bIsRenderingPaused` instead.

Is there a reason for performance?

**Comment by Lpsd** [comment]
There's no reason for a Lua script to be able to modify this value (as it's a property which can be set with `setBrowserProperty`). If anything you don't want that.

This value should only be set via `SetRenderingPaused`

File: Client/cefweb/CWebView.cpp [hunk 1]
```diff
@@ -450,7 +459,8 @@ bool CWebView::GetFullPathFromLocal(SString& strPath)
                 return;
 
             result = m_pEventsInterface->Events_OnResourcePathCheck(strPath);
-    }, this);
+        },
+        this);
```
Comment (Lpsd) [inline Client/cefweb/CWebView.cpp]: Unrelated formatting change. Please revert

File: Client/cefweb/CWebView.cpp [hunk 2]
```diff
@@ -496,7 +506,8 @@ bool CWebView::VerifyFile(const SString& strPath, CBuffer& outFileData)
                 return;
 
             result = m_pEventsInterface->Events_OnResourceFileCheck(strPath, outFileData);
-    }, this);
+        },
+        this);
```
Comment (Lpsd) [inline Client/cefweb/CWebView.cpp]: Unrelated formatting change. Please revert

File: Client/cefweb/CWebView.cpp [hunk 3]
```diff
@@ -816,7 +827,7 @@ bool CWebView::OnBeforeBrowse(CefRefPtr<CefBrowser> browser, CefRefPtr<CefFrame>
 //                                                                //
 ////////////////////////////////////////////////////////////////////
 CefResourceRequestHandler::ReturnValue CWebView::OnBeforeResourceLoad(CefRefPtr<CefBrowser> browser, CefRefPtr<CefFrame> frame, CefRefPtr<CefRequest> request,
-                                                              CefRefPtr<CefRequestCallback> callback)
+                                                                      CefRefPtr<CefRequestCallback> callback)
```
Comment (Lpsd) [inline Client/cefweb/CWebView.cpp]: Unrelated formatting change. Please revert

**Review by Lpsd — CHANGES_REQUESTED** [review]
Looks good to me. Just a few unintentional formatting changes which need reverting and this is ready to go.

**Review by Lpsd — APPROVED** [review]
Thanks for making the requested changes.

lgtm :shipit:

**Pull request merged**